lib/ansible/utils/encrypt.py

```diff
@@ -270,12 +270,13 @@ def _hash(self, secret, salt, salt_size, rounds, ident):
 
 
 def passlib_or_crypt(secret, algorithm, salt=None, salt_size=None, rounds=None, ident=None):
+    display.deprecated("passlib_or_crypt API is deprecated in favor of do_encrypt", version='2.20')
+    return do_encrypt(secret=secret, algorithm=algorithm, salt=salt, salt_size=salt_size, rounds=rounds, ident=ident)
```

Comment:
This function needs to be tested by unit tests. The tests will be removed once this function is removed.

Also, the first two args need to be changed to match `do_encrypt`:

```suggestion
    return do_encrypt(secret, algorithm, salt=salt, salt_size=salt_size, rounds=rounds, ident=ident)
```